pkg/liqo-controller-manager/authentication/forge/identity.go

```diff
@@ -22,24 +22,35 @@ import (
 	"github.com/liqotech/liqo/pkg/consts"
 )
 
+// ControlPlaneIdentityName forges the name of a ControlPlane Identity resource given the remote cluster name.
+func ControlPlaneIdentityName(remoteClusterName string) string {
+	return "controlplane-" + remoteClusterName
+}
+
+// ResourceSliceIdentityName forges the name of a ResourceSlice Identity.
+func ResourceSliceIdentityName(resourceSlice *authv1alpha1.ResourceSlice) string {
+	return "resourceslice-" + resourceSlice.Name
+}
+
 // IdentityForRemoteCluster forges a Identity resource to be applied on a remote cluster.
-func IdentityForRemoteCluster(localClusterIdentity discoveryv1alpha1.ClusterIdentity, identityType authv1alpha1.IdentityType,
+func IdentityForRemoteCluster(name string, localClusterIdentity discoveryv1alpha1.ClusterIdentity, identityType authv1alpha1.IdentityType,
 	authParams *authv1alpha1.AuthParams, namespace *string) *authv1alpha1.Identity {
-	identity := Identity(localClusterIdentity)
+	identity := Identity(name, "")
```

Comment:
The empty parameter here is quite strange. Can we add it as a function parameter?